client/src/modules/templates/bhDateInterval.tmpl.html

```diff
@@ -17,16 +19,16 @@
       </a>
     </li>
 
-    <li class="pull-right" ng-if = "$ctrl.canClear">
+    <li class="pull-right" ng-if="$ctrl.canClear">
       <a href ng-click="$ctrl.clear()" tabindex="-1">
         <i class="fa fa-eraser"></i> <span translate>FORM.BUTTONS.CLEAR</span>
       </a>
     </li>
   </ul>
 
-  <div class="row" date-id="$ctrl.dateId" data-bh-date-interval>
+  <div class="row" date-id="$ctrl.dateId">
```

Comment:
Aha!  Located it - the tests are failing because the [data-bh-date-interval tag is used to locate this element on the page](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/test/end-to-end/shared/components/bhDateInterval.js#L13).  I think if you restore this tag, it should pass tests.